test/codegen/swift/loop_switch/advanced.re

```diff
@@ -0,0 +1,228 @@
+// re2swift $INPUT -o $OUTPUT -cf
+import Foundation
```

Comment:
Sorry I didn't notice it before, but we only add codegen models that are not tested in the main examples (non-default ones) here. We can leave these tests if you think there's some additional value, though.

Reply:
I added all the tests for parity with other backends (and also because I wrote the tests before I wrote the examples). I'm not sure what to do here, should we delete the loop switch tests?

Reply:
Hmm... yes, let's delete them. All these features are covered by examples.

Reply:
(Just the loop/switch tests I mean.)

Reply:
Alrighty, done.